unischedule/lib/routes/go_router_provider.dart

```diff
@@ -27,6 +28,14 @@ final goRouterProvider = Provider<GoRouter>((ref) {
         },
       ),
 
+      GoRoute(
+        path: '/create-class',
+        name: 'create-class',
+        builder: (context, state) {
+          return CreateClassPage(key: state.pageKey);
+        }
+      ),
+
```

Comment:
Can you guys check if this is appropriate? I had to put this route outside of the shell to avoid rendering the bottom navigation bar @SnowArtz @iGotty

Reply:
I think this is appropiate!